techniques/failures/failure-of-222-due-to-blinking.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<title>Failure of Success Criterion 2.2.2 due to using blinking content that lasts for more than five seconds without a mechanism to stop it</title>
+		<link rel="stylesheet" type="text/css" href="../../css/editors.css" class="remove"/>
+	</head>
+	<body>
+		<h1>Failure of Success Criterion 2.2.2 due to using blinking content that lasts for more than five seconds without a mechanism to stop it</h1>
+		<section class="meta">
+			<p class="id">ID: F47</p>
+			<p class="technology">Technology: failures</p>
+			<p class="type">Type: Failure</p>
+		</section>
+		<section id="applicability">
+			<h2>When to Use</h2>
+			<p>Blinking content created using CSS animations or client-side scripting.</p>
+		</section>
+		<section id="description">
+			<h2>Description</h2>
+			<p>Blinking content can be distracting and overwhelming for users, for example people with cognitive disabilities such as attention disorders. Content that starts automatically, blinks for five seconds or more, is presented in parallel with other content, and does not have a method to stop the blinking fails <a href="../../understanding/20/pause-stop-hide.html">Success Criterion 2.2.2</a>.</p>
```

Comment:
minor wording suggestion - including the word "flash" as I've found that to be a common word to describe this as well.  But if you prefer the original text, then please ignore.

```suggestion
			<p>Blinking content can be distracting and overwhelming for users, for example people with cognitive disabilities such as attention disorders. Content that automatically starts to blink or "flash", for five seconds or more, is presented in parallel with other content, and does not have a method to stop the blinking fails <a href="../../understanding/20/pause-stop-hide.html">Success Criterion 2.2.2</a>.</p>
```